### ISSUE TITLE
Embedded browser & open crawled file

### ISSUE THREAD

**Issue by binhlvu** [ViDA-NYU/ache#140]
It's very nice if I can browse the crawling web inside ACHE (localhost:8080) for debugging purpose. Because sometime pages ACHE got are different with pages I get in my browser.

Also, it would save users lot of time to debug the crawled page if we can open it directly in ACHE (without manually writing code to read content and save to html file).

**Comment by aecio** [comment]
I agree, it would be a nice feature. Can you give more details? Would you like to see the cached HTML source code, or maybe the cached HTML rendered in something like an iframe, or the extracted text is sufficient?

**Comment by binhlvu** [comment]
Thank you for considering this.

I think a cached HTML rendered in an iframe is good. Also, if you guy allows users to specify an URL of a cached HTML to render, it would be great.
